src/Robot.h

```diff
@@ -33,13 +43,23 @@ class Robot: public IterativeRobot {
 	virtual void TeleopPeriodic();
 	virtual void TestPeriodic();
 
-private:
 	Scheduler *scheduler;
 	Tester *tester;
-	bool hasRunAuto;
+
+	std::unique_ptr<Claw> claw;
```

Comment:
Is all this junk strictly necessary? Are we really worried some rogue programmer will new up 30 Claws?